packages/block-editor/src/components/use-editor-feature/index.js

```diff
@@ -81,7 +81,18 @@ export default function useEditorFeature( featurePath ) {
 			// such as core/heading or core/post-title.
 			const globalPath = `__experimentalFeatures.global.${ featurePath }`;
 			const blockPath = `__experimentalFeatures.${ blockName }.${ featurePath }`;
-			return get( settings, blockPath ) ?? get( settings, globalPath );
+			const experimentalFeature =
+				get( settings, blockPath ) ?? get( settings, globalPath );
+			if ( experimentalFeature !== undefined ) {
+				return experimentalFeature;
+			}
+
+			// 3 - Fall back to block support.
+			// This is only necessary to support typography.dropCap.
+			// when __experimentalFeatures are not present (core without plugin).
+			// To remove when __experimentalFeatures are ported to core.
+			const { getBlockSupport } = select( 'core/blocks' );
+			return getBlockSupport( blockName, featurePath );
```

Comment:
We only want this for typography.dropCap but in the end, we are adding a mechanism that allows to control any feature not explicitly set on theme.json via block.json. People may start relying on this and then removing the behavior would be breaking backward compatibility.

In practice what happens is that on WordPress 5.6 there is no way a theme or plugin can disable the dropCap right? It seems to be always true without a way to change that. If that's case could our fallback be as simple as returning true all the time if featurePath is typography.dropCap?
We would have:
```
// 3 - Fall back for "typography.dropCap".
if( featurePath === 'typography.dropCap' ) {
	return true;
}
```

Reply:
Implemented this at https://github.com/WordPress/gutenberg/pull/25979/commits/62ac47f84d038f6087a758da134fc208583443d4

Reply:
@jorgefilipecosta @nosolosw the below is true as far as I can see now that 5.6 is released.

> In practice what happens is that on WordPress 5.6 there is no way a theme or plugin can disable the dropCap right

Is there any way to disable dropCap? I understand the rationale behind this PR, but removing functionality that people are actively using while providing no alternative is not particularly useful.